File: src/main/java/com/github/caijh/commons/util/RespBodyNoOpConvertor.java
package com.github.caijh.commons.util;

public class RespBodyNoOpConvertor implements RespBodyConvertor<String> {

    public static final RespBodyNoOpConvertor INSTANCE = new RespBodyNoOpConvertor();

    @Override
    public String convert(String respBody) {
        return respBody;
    }

}

File: src/main/kotlin/com/github/caijh/commons/util/RespBodyNoOpConvertor.kt
package com.github.caijh.commons.util

class RespBodyNoOpConvertor : RespBodyConvertor<String?> {
    override fun convert(respBody: String?): String {
        return respBody!!
    }

    companion object {
        val INSTANCE = RespBodyNoOpConvertor()
    }
}
